pull/context.go

```diff
@@ -30,6 +30,9 @@ type MembershipContext interface {
 
 	// IsCollaborator returns true if the user meets the desiredPerm of the given organzation's repository.
 	IsCollaborator(org, repo, user, desiredPerm string) (bool, error)
+
+	// CollaboratorPermission returns the permission of the user for the given oranization's repository.
+	CollaboratorPermission(org, repo, user string) (string, error)
```

Comment:
I'm going to revert this addition to the interface tomorrow. I don't think its necessary now that I'm using the graphql endpoint